testing/smoke-ide-test/src/smokeIdeTest/groovy/org/gradle/ide/sync/AbstractIdeSyncTest.groovy

```diff
@@ -31,16 +31,18 @@ import java.nio.file.Path
 import java.nio.file.Paths
 
 /**
- * Tests that runs a project import to IDEA, with an provisioning of the desired version.
+ * Tests that runs a project import to IDE, with an provisioning of the desired version.
  *
- * Provisioned IDEs are cached in the {@link AbstractIdeaSyncTest#getIdeHome} directory.
+ * Provisioned IDEs are cached in the {@link AbstractIdeSyncTest#getIdeHome} directory.
  */
 @Timeout(600)
 @CleanupTestDirectory
-abstract class AbstractIdeaSyncTest extends Specification {
+abstract class AbstractIdeSyncTest extends Specification {
```

Comment:
I am possibly lacking context, but since Android Studio is based on IDEA, I didn't see a problem with the previous name (also, other IDEs are not supported, only IDEA and some other IDEA-based IDE).